### PR TITLE
EOS-19535: Added prepare command

### PR DESCRIPTION
**Pull request by indrajitzagade**
## Problem Statement
<pre>
  <code>
  Story Ref (if any):
    Add mini-provisioner prepare command.
  </code>
</pre>
## Unit testing on RPM done
<pre>
  <code>
  Yes
  </code>
</pre>
## Problem Description
<pre>
  <code>
    The mini-provisioner prepare command is missing.
  </code>
</pre>
## Solution
<pre>
  <code>
    Added mini-provisioner prepare command.
  </code>
</pre>
## Unit Test Cases
<pre>
  <code>
    Successfully tested with single node by running the new command.
  </code>
</pre>

### CODE REVIEW

**Review by ajay-paratmandali — APPROVED** [review]

File: conf/mini_provisioner/v2/ha_setup.yaml [hunk 1]
```diff
@@ -18,6 +18,10 @@ ha:
         cmd: ha_setup post_install
         args: --config $URL
 
+    prepare:
```
Comment (ajay-paratmandali) [inline conf/mini_provisioner/v2/ha_setup.yaml:21]: can you remove ha_setup.yaml 
covering in https://github.com/Seagate/cortx-ha/pull/373/files

Comment (indrajitzagade) [reply conf/mini_provisioner/v2/ha_setup.yaml:21]: sure

Comment (indrajitzagade) [reply conf/mini_provisioner/v2/ha_setup.yaml]: done @ajay-paratmandali

**Review by shindeam — APPROVED** [review]
LGTM

**Review by azheregelya — APPROVED** [review]

**Pull request merged**